### ISSUE TITLE
Installer Error

### ISSUE THREAD

**Issue by jpb101090** [Filigrani/SkyCoop#77]
**Mod Version:** v2.1  
**Game Version:** v2.02  
**Launcher:** Steam 
**Additional mods installed:** No  
**Perspective:** Client

### Description
I am attempting to install the installer for Sky-co-op however when I attempt to this error pops up, then it brings me to the download screen for the installer once again.
![image_2024-06-09_163007731](https://github.com/Filigrani/SkyCoop/assets/5152359/1090709c-8824-45f1-ba48-26d3b91d9446)

### How to reproduce
No idea how to reproduce this error.

### Expected behavior
I expected the installer to well launch. I am attempting to play it with someone else and they were able to install it perfectly fine.

### Logs
 N/A no logs

**Comment by KibblesTheKitten** [comment]
Is your internet connection stable and can it handle loads of approximately 30-50mbps? If it can then it's just being annoying

**Comment by jpb101090** [comment]
Yes, my internet connection is stable and is able to handle loads of 30-50mbps or higher.

**Comment by KibblesTheKitten** [comment]
Weird. Your ISP is not blocking GitHub, so maybe it's just a proxy on your device or something won't let it access the internet?

**Comment by jpb101090** [comment]
sooo I don't know how to set up or remove a proxy. Though I know it's not a ISP issue or anything cause it does work on another pc. It's just that, that one is annoying to play on.
